plsql-checks/src/test/resources/checks/to_date_without_format.sql

```diff
@@ -1,7 +1,7 @@
 begin
   mydate := to_date('2015-01-01'); -- Noncompliant {{Specify the date format in this TO_DATE.}}
 --          ^^^^^^^^^^^^^^^^^^^^^
-  
+  to_date(null);
```

Comment:
Since Oracle doesn't differentiate between NULL and a zero length string, you could add a test for to_date('') too.